decode.go

```diff
@@ -851,7 +851,10 @@ func (d *Decoder) decodeStruct(dst reflect.Value, src ast.Node) error {
 						continue
 					}
 					fieldName := fieldErr.StructField()
-					structField := structFieldMap[fieldName]
+					structField, ok := structFieldMap[fieldName]
+					if !ok {
```

Comment:
This library uses `exists` as variable name instead of `ok` when checking the existence of map key .